examples/code/command-line-parsing/md5_succinct/md5.ml

```diff
@@ -12,7 +12,7 @@ let command =
   Command.basic
     ~summary:"Generate an MD5 hash of the input data"
     ~readme:(fun () -> "More detailed information")
-    Command.Param.(
+    (Command.Param.(
```

Comment:
Better I think to remove the paren at the end. The outer parens are not necessary.